src/HomeComponent/HomeComponent.css

```diff
@@ -85,296 +89,287 @@
   display: flex;
   flex-direction: row;
 }
+/* week 1 */
+.week-container.week-container-one{
 
-.week-container1 .exercise-link-completed {
-  background-color: rgba(255,178,0,0.85);
+  margin-top: 25px;
+  background-image: linear-gradient(to left, rgba(255,233,220,0), rgb(255,209,103,0.85));;
 }
 
-.week-container1 .exercise-link-progress {
-  border: 1px solid red;
+.week-container.week-container-one.size0{
```

Comment:
@notej-k You can have the following classes that contain the style for background size

`.size1 {
  background-size: 50% 100%;
}

.size2 {
  background-size: 75% 100%;
}

.size3 {
  background-size: 100% 100%;
}`

You can add ` background-repeat: no-repeat;` to `week-container`

You can have the specific week container style separated for week:

`.week-container-one{
  margin-top: 25px;
  background-image: linear-gradient(to left, rgba(255,233,220,0), rgb(255,209,103,0.85));;
}`

Then you can refer the classes as e.g for week one:
`<div className={'week-container week-container-one size' + weekStatus(1)}>`

Please keep in mind, whenever you have similar code that can be reused follow this kind of approach :)